person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    is_of_age? || @parent_permission == true
+  end
```

Comment:
- [ ] Kindly change the `is_of_age?` to `of_age?` as the name of the private method is now `of_age?`.